bandpass/README.md

```diff
@@ -122,31 +132,46 @@ Run these commands in the directory containing `p_lowpass.py`:
         AS LANGUAGE 'Python' NAME 'p_lowpass_factory' 
         LIBRARY :library fenced;
 
+The first two lines define some symbols used in the subsequent lines. 
```

Comment:
Nice addition!  This part looks much better now.

(Nit: "set up" for a verb, "setup" for an adjective or noun.)